packages/app/src/app/components/Title/index.tsx

```diff
@@ -1,14 +1,18 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import delayEffect from '@codesandbox/common/lib/utils/animation/delay-effect';
 
-export default styled.h1`
-  ${props => props.delay != null && delayEffect(props.delay || 0)};
-  color: white;
-  font-size: 2.5rem;
-  font-weight: 300;
-  background-color: transparent;
-  margin-top: 0;
-  border: none;
-  outline: none;
-  text-align: center;
+const Title = styled.h1`
+  ${({ delay = 0 }: { delay: number | null }) => css`
```

Comment:
@Saeris I think it's better to extract the `TitleProps` here, just to be consistent